### ISSUE TITLE
Version of RE2 being wrapped?

### ISSUE THREAD

**Issue by adamhamlin** [andreasvc/pyre2#34]
This is not a proper issue, just a question about which version of re2 is being wrapped.

Based on some log statements that appeared, my guess is that the re2 version is `2016-XX-XX` or older, but I'm not positive. Where in the code is that specified or computed?

Thanks

**Comment by andreasvc** [comment]
If you compile from source, it uses the version you have installed. If you use e.g. the Linux wheel, it has been compiled against the libre2-dev version on Ubuntu 20.04.

Why do you ask?

**Comment by adamhamlin** [comment]
I am trying to use the latest pypi package (version `0.3.6`), but noticed some different behavior when using it on macos vs linux on docker. Specifically, seeing this INFO log but only on docker: https://github.com/google/re2/blob/78dd4fa1f86bafbf5a5eb006778d9e6e27297af6/re2/dfa.cc#L472

That's re2 version `2016-09-01`, but that log message changes/moves on later versions, so mine must be using `2016-09-01` or earlier.

Am I missing something? Or Is there no packaged version that's using a more recent re2 build?

**Comment by sarnold** [comment]
You are right that the current manylinux wheels are built with a (now)deprecated version, including a really old version of the google-y library.  Among other updates, we'll be updating the platform wheel builds along the way.  Thanks for the report!

**Comment by sarnold** [comment]
There is now an open PR #47 that was the latest cibuildwheel that includes the latest quay.io docker image with a newer CentOS and re2.  After this gets addressed and I have my next "free" time I can look at vendoring the google-y source bits for the platform wheels.  In the mean time you can use a tox env to use native build BUT you would need to install the build deps (mainly re2 and pybind11) before building with tox.
